db/migrate/20220106181004_add_status_to_items.rb

```diff
@@ -0,0 +1,5 @@
+class AddStatusToItems < ActiveRecord::Migration[5.2]
+  def change
+    add_column :items, :status, :integer, default: 0
```

Comment:
this migration will require we update the item factories to make a default value. Have you added validations to the item model to require the status?

Reply:
i see you added the default to the factory. nice!

Reply:
I see you added an enum for the item status. How do we validate enums? Is declaring one enough or do we need to add something? I totally forget.

Reply:
> I see you added an enum for the item status. How do we validate enums? Is declaring one enough or do we need to add something? I totally forget.

`describe 'enum validation' do
    it { should define_enum_for(:status).with([:in_progress, :cancelled, :completed])}
  end
`

An example from `invoice_items` of how to test your enums. IIRC, Jamison in the checkin said that the enum sort of validates itself. It will only accept/recognize intergers that it has (In the above example, 0-2). Anything outside of those parameters, the enum won't allow it to persist to the database.